app/controllers/posts_controller.rb

```diff
@@ -12,6 +9,14 @@ def index
     @posts = Post.all
```

Comment:
I think the sequence is index, show, new, edit, create, update, destroy and then the private methods.